### ISSUE TITLE
Manifest request is timed out, and storybook freezes when I change the my codes.

### ISSUE THREAD

**Issue by yas-tyoukan** [storybookjs/storybook#10182]
**Describe the bug**

When starting storybook and I change my code, It's not reflect and I can't reload browser. And the following error occur in console.

```
Uncaught (in promise) Error: Manifest request to 89ffd3dc8096c1c7d5e1.hot-update.json timed out.
    at XMLHttpRequest.request.onreadystatechange (runtime~main.5795d55f1405382b4c86.bundle.js:sourcemap:92)
```

I try pure enviroment, but it reproduce.

**To Reproduce**
1. ` mkdir mydir && cd mydir && npm init -y && npx -p @storybook/cli sb init --type react && npm run storybook`
2. I change `mydir/stories/1-Button.stories.js` 'Hello Button' to 'Hello Good Button'.
3. The changes is not reflect in browser and page reload is not work.
4. The following error occurs in browser console.

```
Uncaught (in promise) Error: Manifest request to 89ffd3dc8096c1c7d5e1.hot-update.json timed out.
    at XMLHttpRequest.request.onreadystatechange (runtime~main.5795d55f1405382b4c86.bundle.js:sourcemap:92)
```

**Comment by yas-tyoukan** [comment]
package.json (generated)

```
{
  "name": "2-1-1",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "storybook": "start-storybook -p 6006",
    "build-storybook": "build-storybook"
  },
  "keywords": [],
  "author": "",
  "license": "ISC",
  "devDependencies": {
    "@babel/core": "^7.9.0",
    "@storybook/addon-actions": "^5.3.17",
    "@storybook/addon-links": "^5.3.17",
    "@storybook/addons": "^5.3.17",
    "@storybook/cli": "^5.3.17",
    "@storybook/react": "^5.3.17",
    "babel-loader": "^8.1.0"
  },
  "dependencies": {}
}
```

**Comment by shilman** [comment]
Do you have a repro repo?

**Comment by yas-tyoukan** [comment]
Sorry,  I create repo https://github.com/yas-tyoukan/react-storybook-test. but, it's not reproduced.

I don't know now why the cause was reproduced, may be my PC slow or may be something unfortunate thing happened.

I close this issue. I apologize for the trouble.

**Comment by christopher-francisco** [comment]
I have this exact same issue on my storybook build. Happened after upgrading to Node 14 and TS 4.3.2

**Comment by Novolokov** [comment]
very annoying, have to restart storybook on every change more or less. I don't think reproduction is possible, just have a gigantic project with typescript

**Comment by kubijo** [comment]
The issue certainly exists, and I am now being constantly annoyed with it... 
I have upgraded to SB 8 in hopes it would resolve this, but no dice.

I wouldn't even know where to start looking and what information would be usefull here.

These are message that I am getting:

```
<w> [webpack.cache.PackFileCacheStrategy] Skipped not serializable cache item 'Compilation/modules|/dev/.yarn/__virtual__/babel-loader-virtual-37cc9b4568/0/cache/babel-loader-npm-9.1.3-cbf4da21df-7086e67827.zip/node_modules/babel-loader/lib/index.js??ruleSet[1].rules[1].use[0]!/dev/packages/aaa/src/proto/mock.tsx': No serializer registered for ConstDependency
<w> while serializing webpack/lib/cache/PackFileCacheStrategy.PackContentItems -> webpack/lib/NormalModule -> Array { 13 items } -> ConstDependency
<w> [webpack.cache.PackFileCacheStrategy] Skipped not serializable cache item 'Compilation/modules|/dev/.yarn/__virtual__/babel-loader-virtual-37cc9b4568/0/cache/babel-loader-npm-9.1.3-cbf4da21df-7086e67827.zip/node_modules/babel-loader/lib/index.js??ruleSet[1].rules[1].use[0]!/dev/packages/bbb/lib/helmet.tsx': No serializer registered for ConstDependency
<w> while serializing webpack/lib/cache/PackFileCacheStrategy.PackContentItems -> webpack/lib/NormalModule -> Array { 10 items } -> ConstDependency
<w> [webpack.cache.PackFileCacheStrategy] Skipped not serializable cache item 'Compilation/modules|/dev/.yarn/__virtual__/babel-loader-virtual-37cc9b4568/0/cache/babel-loader-npm-9.1.3-cbf4da21df-7086e67827.zip/node_modules/babel-loader/lib/index.js??ruleSet[1].rules[1].use[0]!/dev/packages/ccc/src/client/store.tsx': No serializer registered for ConstDependency
<w> while serializing webpack/lib/cache/PackFileCacheStrategy.PackContentItems -> webpack/lib/NormalModule -> Array { 18 items } -> ConstDependency
<w> [webpack.cache.PackFileCacheStrategy] Skipped not serializable cache item 'Compilation/modules|/dev/.yarn/__virtual__/babel-loader-virtual-37cc9b4568/0/cache/babel-loader-npm-9.1.3-cbf4da21df-7086e67827.zip/node_modules/babel-loader/lib/index.js??ruleSet[1].rules[1].use[0]!/dev/packages/ddd/lib/scalar/getScalarFetcher.tsx': No serializer registered for ConstDependency
<w> while serializing webpack/lib/cache/PackFileCacheStrategy.PackContentItems -> webpack/lib/NormalModule -> Array { 8 items } -> ConstDependency
<i> [webpack-dev-middleware] wait until bundle finished: /shared_lib_net_index_tsx.iframe.bundle.js.map
<i> [webpack-dev-middleware] wait until bundle finished: /vendors-_yarn___virtual___connectrpc-connect-virtual-e5a63cf55b_0_cache_connectrpc-connect-np-116801.iframe.bundle.js.map
<i> [webpack-dev-middleware] wait until bundle finished: /vendors-_yarn___virtual___connectrpc-connect-virtual-e5a63cf55b_0_cache_connectrpc-connect-np-547b50.iframe.bundle.js.map
<i> [webpack-dev-middleware] wait until bundle finished: /runtime~main.iframe.bundle.js.map
<i> [webpack-dev-middleware] wait until bundle finished: /packages_toolbox_src_proto_mock_tsx.iframe.bundle.js.map
<i> [webpack-dev-middleware] wait until bundle finished: /toolbox-src-app-pages-Devices-devices-stories.iframe.bundle.js.map
<w> [webpack.cache.PackFileCacheStrategy] Serializing big strings (125kiB) impacts deserialization performance (consider using Buffer instead and decode when needed)
<w> [webpack.cache.PackFileCacheStrategy] Serializing big strings (202kiB) impacts deserialization performance (consider using Buffer instead and decode when needed)
<w> [webpack.cache.PackFileCacheStrategy] Serializing big strings (137kiB) impacts deserialization performance (consider using Buffer instead and decode when needed)
<w> [webpack.cache.PackFileCacheStrategy] Serializing big strings (188kiB) impacts deserialization performance (consider using Buffer instead and decode when needed)
<w> [webpack.cache.PackFileCacheStrategy] Serializing big strings (173kiB) impacts deserialization performance (consider using Buffer instead and decode when needed)
<w> [webpack.cache.PackFileCacheStrategy] Serializing big strings (182kiB) impacts deserialization performance (consider using Buffer instead and decode when needed)
<w> [webpack.cache.PackFileCacheStrategy] Serializing big strings (100kiB) impacts deserialization performance (consider using Buffer instead and decode when needed)
<w> [webpack.cache.PackFileCacheStrategy] Serializing big strings (103kiB) impacts deserialization performance (consider using Buffer instead and decode when needed)
<w> [webpack.cache.PackFileCacheStrategy] Serializing big strings (152kiB) impacts deserialization performance (consider using Buffer instead and decode when needed)
```

**Comment by kubijo** [comment]
I have tried a lot ... changing settings of webpack caching in many ways (didn't help), temporarily changing stories patterns to limit the number of stories (that broke it completely with no sense in sight)

**Comment by kubijo** [comment]
I have spent quite a lot of time trying to first get rid of the errors posted above (`webpack.cache.PackFileCacheStrategy`).

I did identify the reason for it (`webpack.ProvidePlugin`) and had a change that would be a base for what would allow me to exclude it from the storybook webpack config.

It did NOT however solve anything with the freezing of Storybook even though I did not have any errors shown at that stage.

Michael Shilman (@shilman): Pinging you so that I know that somebody sees this and perhaps can say where do we go from this.

I really tried a lot and wasn't able to identify anything that would give me hints as to what was the problem and how to fix this.

If it works (for at least a while) it's a pleasure to work with.
When it doesn't work, I have to practically just work speculatively without hot reloaded preview because otherwise I have to kill Storybook and start it again on every change.
